mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -2707,7 +2707,10 @@ def Vector_MatmulOp : Vector_Op<"matrix_multiply", [Pure,
     <rhs_columns> and multiplies them. The result matrix is returned embedded in
     the result vector.
 
-    Also see:
+    Note, the semantics of the corresponding LLVM intrinsic,
+    `@llvm.matrix.multiply.*`, are not clear in the context of scalable
+    vectors. Hence, this Op is only available for fixed-width vectors. Also
+    see:
```

Comment:
I just had a quick look at https://llvm.org/docs/LangRef.html#matrix-intrinsics and noticed
> These intrinsics require matrix dimensions to be passed as immediate arguments

So the shape can't represent scalable quantities. I think we can conclude these intrinsics don't support scalable vectors?

Reply:
Yup,  thanks for checking 👍🏻